Transmission.xcodeproj/project.pbxproj

```diff
@@ -3621,14 +3619,17 @@
 				CODE_SIGN_IDENTITY = "-";
 				ENABLE_HARDENED_RUNTIME = YES;
 				FRAMEWORK_SEARCH_PATHS = macosx;
-				GCC_PREFIX_HEADER = macosx/Transmission_Prefix.pch;
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
 					.,
 				);
 				LD_RUNPATH_SEARCH_PATHS = "@executable_path/../Frameworks";
 				OTHER_CFLAGS = "$(inherited)";
-				OTHER_CPLUSPLUSFLAGS = "$(OTHER_CFLAGS)";
+				OTHER_CPLUSPLUSFLAGS = (
```

Comment:
Why not `CLANG_ENABLE_MODULES = YES;`?

Reply:
<img width="607" alt="image" src="https://user-images.githubusercontent.com/8330119/174485515-52d67fb8-e0aa-4805-a3fd-a96e846ed141.png">

Reply:
I believe this setting doesn't support ObjC++ modules

Reply:
I can check it later